ros2_generate_interface_docs/ros2_generate_interface_docs/main.py

```diff
@@ -23,11 +23,45 @@
 from ros2_generate_interface_docs import srv_utils
 from ros2_generate_interface_docs import utils
 
+from rosidl_runtime_py import get_action_interfaces
 from rosidl_runtime_py import get_message_interfaces
 from rosidl_runtime_py import get_service_interfaces
 
 
-def generate_interfaces(interfaces, html_dir, template, interface_type):
+def generate_interfaces_index(messages, services, actions, html_dir, timestamp):
+    """
+    Generate index for packages.
+
+    :param messages: Dictionay with the message package as a key and a list message names as value
```

Comment:
Spelling: Dictionay. Also the type is already documented by `:param type: dict` so I would suggest to not repeat it in the text.

Same below. Similar also for `list`.